Formula/go-jira.rb

```diff
@@ -0,0 +1,37 @@
+class GoJira < Formula
+  desc "Simple jira command-line client in Go"
+  homepage "https://github.com/Netflix-Skunkworks/go-jira"
+  url "https://github.com/Netflix-Skunkworks/go-jira/archive/v1.0.12.tar.gz"
+  sha256 "7ce5d6dcc5021f2019b0107c4ad342979d18bb110188b810017b2d94ff384a33"
+
+  bottle do
+    cellar :any_skip_relocation
+    sha256 "f60aa1ae0c9334d49b61adbf0e3b222076f07b975186bcac36d1583999265a5d" => :sierra
+  end
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    dir = buildpath/"src/gopkg.in/Netflix-Skunkworks/go-jira.v1"
+    dir.install buildpath.children
+
+    ldflags = ["-w", "-s"]
+    cd dir do
+      system "go", "build", "-o", bin/"jira", "-ldflags", ldflags.join(" "), "cmd/jira/main.go"
+    end
+  end
+
+  test do
+    # `test do` will create, run in and delete a temporary directory.
+    #
+    # This test will fail and we won't accept that! For Homebrew/homebrew-core
+    # this will need to be a test that verifies the functionality of the
+    # software. Run the test with `brew test go-jira`. Options passed
+    # to `brew install` such as `--HEAD` also need to be provided to `brew test`.
+    #
+    # The installed folder is not in the path, so use the entire path to any
+    # executables being tested: `system "#{bin}/program", "do", "something"`.
+    system "#{bin}/jira", "version"
```

Comment:
We require new submissions to homebrew-core to have a more detailed test than just `version` or `help`. Check out [cmake](https://github.com/Homebrew/homebrew-core/blob/b5b1add9526f40a3f354124082a553bbc9b560c0/Formula/cmake.rb) for an example of a good test.

Reply:
Hm, didn't know that. If your build process tosses the file system afterward I can set it to `jira export-templates` instead, that should invoke actual functionality without requiring a running Atlassian Cloud instance.

Reply:
> If your build process tosses the file system afterward

It does, yep.